pysdsl/test/test_sorted_int_stack.py

```diff
@@ -0,0 +1,52 @@
+from pysdsl import SortedIntStack as stack
+import random
+
+def test_push_pop_top_empty():
+    s = stack(max_value=1000)
+    a = []
+
+    assert(s.empty() and s.size == 0 and len(s) == 0)
+
+    a = list(sorted(set([random.randint(0, 999) for _ in range(1000)])))
```

Comment:
sorted always returns list. instead of `set([... for _ in ...])` you can just use `{... for _ in ...}`